% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{book}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={97039 - GLOBAL HEALTH, ANTIMICROBIAL DRUGS AND VACCINES},
  pdfauthor={Russell E. Lewis, Associate Professor, Infectious Diseases, University of Bologna},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newlength{\cslentryspacingunit} % times entry-spacing
\setlength{\cslentryspacingunit}{\parskip}
\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1
  \let\oldpar\par
  \def\par{\hangindent=\cslhangindent\oldpar}
  \fi
  % set entry spacing
  \setlength{\parskip}{#2\cslentryspacingunit}
 }%
 {}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi

\title{97039 - GLOBAL HEALTH, ANTIMICROBIAL DRUGS AND VACCINES}
\usepackage{etoolbox}
\makeatletter
\providecommand{\subtitle}[1]{% add subtitle to \maketitle
  \apptocmd{\@title}{\par {\large #1 \par}}{}{}
}
\makeatother
\subtitle{The Crisis in Antibiotic Development}
\author{Russell E. Lewis, Associate Professor, Infectious Diseases, University of Bologna}
\date{11/26/2022}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\hypertarget{the-growing-crises-of-antibiotic-development}{%
\section*{The growing crises of antibiotic development}\label{the-growing-crises-of-antibiotic-development}}
\addcontentsline{toc}{section}{The growing crises of antibiotic development}

In module 1, we discussed the history of antibiotic discovery and the current challenges with antimicrobial resistance that has been amplified by the lack of development of new antibiotics. In this module, we will examine the scientific and economic challenges associated with antibiotic development, and compare and contrast strategies that have been proposed to stimulate development of new antibiotics.

As we discussed previously, antibiotic discovery began to slow in the 1980's leading to a discovery void in new molecular entities (NMEs) since the 1990s. This lack of antibiotic innovation occurred at a critical period when antibiotic resistance, particularly to many front-line beta-lactam antibiotics began to increase rapidly due to the emergence and worldwide diffusion of new forms of enzymatic (beta-lactamase) resistance to antibiotics. These bacterial enzymes can be broadly classified as:

\begin{itemize}
\tightlist
\item
  Narrow-spectrum beta-lactamases, which act on penicillins and first-generation cephalosporins (e.g., TEM-1 and 2, SHV-1, cephalosporinases, OXA-type enzymes
\item
  Extended-spectrum beta-lactamases (ESBLs), which act on penicillins and all four generations of cephalosporins (SHV-2, SHV-5, SHV-7, SHV-12, TEM-10, TEM-12, TEM-26, CTX-M, OXA-type ESBLs)
\item
  Carbapenemases, which act on penicillins, all four generations of cephalosporins, and carbapenems (KPC, NDM-1, VIM and IMP carbapenemases, OXA-type carbapenemases).
\end{itemize}

These beta-lactamases are problematic because they have a low barrier for creating resistance to multiple agents that are considered first and second line therapies of choice for common infection. Indeed, often a single amino acid changes in the enzymes can change their ability to hydrolyze new antibiotics. Additionally, many of these enzymes are encoded on spread on plasmids (mobile genetic elements that can be passed from one bacterial species to another) resulting in rapid dissemination of resistance.

\includegraphics[width=5.20833in,height=\textheight]{images/betalactamases.png}

\textbf{Figure 1. Increase in numbers of group 1, 2, and 3 beta-lactamases from 1970 to 2009}. Figure is from Bush and Jacoby\textsuperscript{1}

A major concern is the increase in carbapenem-resistant Gram-negative pathogens encoded by a range of genes, including NDM-1, OXA-48, KPC, VIM, and IMP. Resistance has rapidly spread globally and across bacterial species since the first reported case in \emph{Klebsiella pneumoniae} less 6 than a decade ago. The emergence of plasmid-mediated genes (\emph{mcr}-1, \emph{mcr}-2, \emph{mcr}-3, \emph{mcr}-4, \emph{mcr}-5, 7 and \emph{icr}-Mo ), which encode resistance to colistin- a drug of last resort, was first reported in 2016 from \emph{Escherichia coli} cultured from a pig in China, but it has since been reported globally and in numerous bacterial species, including \emph{K. pneumoniae} and \emph{Enterobacter} species. Additionally, multiple variants of genes encoding extended-spectrum beta-lactamase (ESBL) and conferring resistance to penicillin, cephalosporins, and monobactam have now spread globally.

Reports from major surveillance networks, including \href{https://atlas.ecdc.europa.eu/public/index.aspx}{EARS-Net in Europe}, indicate high prevalence in \emph{E. coli} and \emph{K. pneumoniae}. ESBL-positive Gram-negative bacteria are found in livestock, meat products, and companion animals. In \emph{Neisseria gonorrhoeae}, resistance to the recommended first-line therapy (azithromycin and ceftriaxone) has been reported in the United Kingdom and Australia with the UK isolate showing resistance for all currently recommend antibiotics.

In early 2017, the WHO convened a group of experts to prioritize the need for new drugs to treat antibiotic- resistant bacteria. The WHO assigned the highest priority to antibacterial drug research and development for the Gram- negative bacteria \emph{Acinetobacter}, \emph{Pseudomonas} and species of \emph{Enterobacterales} that are resistant to carbapenems and are usually extensively drug resistant (XDR).

\textbf{Table 1. WHO priority pathogens}

\begin{longtable}[]{@{}
  >{\raggedright\arraybackslash}p{(\columnwidth - 2\tabcolsep) * \real{0.1389}}
  >{\raggedright\arraybackslash}p{(\columnwidth - 2\tabcolsep) * \real{0.8611}}@{}}
\toprule
\begin{minipage}[b]{\linewidth}\raggedright
Priority
\end{minipage} & \begin{minipage}[b]{\linewidth}\raggedright
Pathogens included
\end{minipage} \\
\midrule
\endhead
\textbf{Critical} & \emph{Acinetobacter baumannii} (Carbapenem-resistant)

\emph{Pseudomonas aeruginosa} (Carbapenem-resistant)

Enterbacterales (3rd generation cephalosporin, carbapenem-resistant) \\
\textbf{High} & \emph{Enterococcus faecium}, vancomycin-resistant

\emph{Staphylococcus aureus}, methicillin-resistant, vancomycin intermediate and resistant

\emph{Helicobacter pylori}, clarithromycin-resistant

\emph{Campylobacter}, fluoroquinolone-resistant

\emph{Salmonella} spp., fluoroquinolone-resistant

\emph{Neisseria gonorrhoeae}, 3rd generation cephalosporin-resistant, fluoroquinolone-resistant \\
\textbf{Medium} & \emph{Streptococcus pneumoniae}, penicillin-non-susceptible

\emph{Haemophilus influenzae}, ampicillin-resistant

\emph{Shigella} spp., fluoroquinolone-resistant \\
\bottomrule
\end{longtable}

The same year, the WHO released a \href{https://apps.who.int/iris/bitstream/handle/10665/330420/9789240000193-eng.pdf}{clinical pipeline report}, which was updated in 2018 and 2019 The clinical pipeline reports analysed antibiotics and biologics according to their activity against the critical priority pathogens carbapenem resistant \emph{Acinetobacter baumannii} (CRAB), carbapenem-resistant \emph{Pseudomonas aeruginosa} (CRPA), extended spectrum beta-lactamase (ESBL) producing Enterobacterales and carbapenem resistant Enterobacterales (CRE). The level of innovation in the global clinical pipeline was assessed on the basis of the absence of pre-existing cross-resistance to currently used antibacterial drugs. The key findings from this report included:

\begin{quote}
\begin{itemize}
\item
  The clinical pipeline remains insufficient to tackle the challenge of increasing emergence and spread of antimicrobial resistance.
\item
  It is primarily driven by small- or medium-sized enterprises (SMEs), with large pharmaceutical companies continuing to exit the field.
\item
  Eight new antibacterial agents have been approved since 1 July 2017, but overall, they have limited clinical benefits.
\item
  One new anti-tuberculosis (anti-TB) agent, pretomanid, developed by a not-for-profit organization, has been approved for use within a set drug-combination treatment for MDR TB.
\item
  The current clinical pipeline contains 50 antibiotics and combinations (with a new therapeutic entity) and 10 biologicals, of which 32 antibiotics are active against the WHO priority pathogens:

  \begin{itemize}
  \item
    Six of these agents fulfil at least one of the innovation criteria; only two of these are active against the critical MDR Gram-negative bacteria.
  \item
    More than 40\% of the pipeline targeting WHO priority pathogens consists of additional β-lactam and β-lactamase inhibitor (BLI) combinations, with a major gap in activity against metallo-βlactamase (MBL) producers.
  \item
    The anti-TB and C. difficile antibacterial pipeline is more innovative than the WHO priority pathogens pipeline, with more than half of the antibiotics fulfilling all of the innovation criteria.
  \end{itemize}
\end{itemize}
\end{quote}

\hypertarget{why-has-antibiotic-discovery-faltered}{%
\section*{Why has antibiotic discovery faltered?}\label{why-has-antibiotic-discovery-faltered}}
\addcontentsline{toc}{section}{Why has antibiotic discovery faltered?}

\hypertarget{scientific-challenges}{%
\subsection*{Scientific challenges}\label{scientific-challenges}}
\addcontentsline{toc}{subsection}{Scientific challenges}

Antibiotic discovery is has special challenges. Antibiotics must attack multiple target species that change over time, by developing resistance, and must do this in multiple body compartments.\textsuperscript{2} The developer of a new antibiotic must guess what resistance problems will be a greater problem in 5-10 years, and develop the drug to overcome this problem that still isn't fully understood. An This flexibility and risk is not required in other therapeutic areas such as hypertension, diabetes, Alzheimer's disease where the drugs bind to one specific target. Even for cancer chemotherapy, which develops resistance to therapy, is not transmissible to other cancers or patients.Antibiotics must also be remarkable non-toxic, as their daily dosages often measured in grams are often higher that for other pharmaceutics measured in milligrams.

Nearly all of the antibiotics used today belong to classes of drugs discovered before 1970. They are products of a ``golden age'' of discovery from 1945-1965, which screened natural products from soil streptomyces and fungi. This discovery approach hit the law of diminishing returns y the 1960's with the same classes (especially tetracyclines) being repeatabley rediscovered.\textsuperscript{2} Since 1970, the only new antibiotic classes to reach the marked are the oxazolidinones (i.e.~linezolid discovered in 1978 launched in 2000) and lipopeptides (discovered in 1986 launched in 2003). Most advances since the 1970s have come through improvements within antibiotic classes yielding analogues with increased potency and greater ability to evade existing resistance. However, over time this approach has become even more difficult with the emergence of resistance given resistance to the entire class of antibiotics such as KPC carbapenemases that compromises nearly allbeta-lactams, 16srRNA methylases that impact all aminoglycosides, and upregulation of efflux pumps that affects multiple antibiotics classes.

Given the sense that the limits of existing antibiotic classes had already been discovered by screening soil organisms, the pharmaceutical industry turned to genomics-based antibiotic discovery in the 1990s.\textsuperscript{2} The approach used genomic sequence data from several pathogens to identify conserved genes encoding targets not found in mammalian cells, and then run high-throughput screens of existing compound libraries to identify ``druggable'' molecules for these targets. Natural product screening was abandoned, partly because it had ceased to identify new leads, was expensive and time consuming, and it fitted poorly with the changing logistics of high-throughput screening.

In summary genomics-based discovery proved notable for its disappointments despite huge early enthusiasm and investment.

From 1995 to 2002, SmithKline Beecham \[now part of GlaxoSmithKline (GSK)\] identified 300 potential targets and ran 67 high-throughput screens, each of 260,000 -- 530,000 compounds.\textsuperscript{3} Sixteen screens led to `hits'---meaning compounds that bound selectively to a target giving a reproducible antimicrobial activity in assays---and five of these translated into `lead' compounds. The five corresponding targets, two (FabI9and Mrs) were not universally essential or conserved, meaning that they could not form the targets of broad-spectrum antibiotics, and it proved impossible to incorporate `drug-like properties' into molecules that bound two others. The final target identified was peptide deformylase, for which GSK now has a molecule (GSK 1322322) in Phase II trials, although this did not come from high-throughput screening. This performance appears typical of other companies that followed the genomics strategy. Thus, 20 years after its advent, no antibiotic developed by this approach has reached the market

\hypertarget{regulatory-approval}{%
\subsection*{Regulatory approval}\label{regulatory-approval}}
\addcontentsline{toc}{subsection}{Regulatory approval}

\hypertarget{clinical-trials}{%
\subsubsection*{Clinical trials}\label{clinical-trials}}
\addcontentsline{toc}{subsubsection}{Clinical trials}

\hypertarget{post-marketing}{%
\subsubsection*{Post-marketing}\label{post-marketing}}
\addcontentsline{toc}{subsubsection}{Post-marketing}

\hypertarget{economic-challenges}{%
\subsection*{Economic challenges}\label{economic-challenges}}
\addcontentsline{toc}{subsection}{Economic challenges}

While antibiotics are typically taken over a short period of time, other drugs for hypertension, cholesterol, diabetes are taken chronically making the economics for development in those therapeutic areas inherently

\hypertarget{what-strategies-can-be-used-to-stimulate-antibiotics-development}{%
\section*{What strategies can be used to stimulate antibiotics development}\label{what-strategies-can-be-used-to-stimulate-antibiotics-development}}
\addcontentsline{toc}{section}{What strategies can be used to stimulate antibiotics development}

Scientific strategies

Economic strategies

\hypertarget{refs}{}
\begin{CSLReferences}{0}{0}
\leavevmode\vadjust pre{\hypertarget{ref-BushJacoby2010}{}}%
\CSLLeftMargin{1. }
\CSLRightInline{Bush, K. \& Jacoby, G. A. \href{https://doi.org/10.1128/AAC.01009-09}{Updated {Functional Classification} of {\(\beta\)}-{Lactamases}}. \emph{Antimicrobial Agents and Chemotherapy} \textbf{54}, 969--976 (2010).}

\leavevmode\vadjust pre{\hypertarget{ref-LivermoreEtAl2011}{}}%
\CSLLeftMargin{2. }
\CSLRightInline{Livermore, D. M. \emph{et al.} \href{https://doi.org/10.1093/jac/dkr262}{Discovery research: The scientific challenge of finding new antibiotics}. \emph{Journal of Antimicrobial Chemotherapy} \textbf{66}, 1941--1944 (2011).}

\leavevmode\vadjust pre{\hypertarget{ref-PayneEtAl2007}{}}%
\CSLLeftMargin{3. }
\CSLRightInline{Payne, D. J., Gwynn, M. N., Holmes, D. J. \& Pompliano, D. L. \href{https://doi.org/10.1038/nrd2201}{Drugs for bad bugs: Confronting the challenges of antibacterial discovery}. \emph{Nature reviews. Drug discovery} \textbf{6}, 29--40 (2007).}

\end{CSLReferences}

\end{document}
